components/dashboard/src/teams/SSO.tsx

```diff
@@ -100,18 +104,12 @@ function OIDCClients(props: { organizationId: string }) {
             </div>
 
             {clientConfigs.length === 0 && (
-                <div className="w-full flex h-80 mt-2 rounded-xl bg-gray-100 dark:bg-gray-900">
-                    <div className="m-auto text-center">
-                        <h3 className="self-center text-gray-500 dark:text-gray-400 mb-4">No OIDC Clients</h3>
-                        <div className="text-gray-500 mb-6">
-                            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor
-                            invidunt ut labore et dolore magna aliquyam
-                        </div>
-                        <button className="self-center" onClick={() => setModal({ mode: "new" })}>
-                            New OIDC Client
-                        </button>
-                    </div>
-                </div>
+                <EmptyMessage
+                    title="No OIDC providers"
+                    subtitle="Enter the client ID, client secret, and other information issued by your identity provider, to enable single sign-on for your organization."
```

Comment:
I'm curious what the empty message should be good for. The general point of "enable single sign-on for your organization" is perfect! 

On this view, one cannot enter anything, thus the call to enter details demands too much. 

I think what really is missing here is the mentioning of supported IdPs. 

WDYT of something like this? 
> Enable single sign-on for your organization using an external identity provider (IdP) service that supports the OpenID Connect (OIDC) standard, such as Google.